metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache-2.0'
 description      'Install Chocolatey on Windows'
 long_description 'Installs the Chocolatey package manager for Windows.'
-version          '2.0.1'
+version          '2.0.2'
```

Comment:
I wouldn't make this adjustment.

Reply:
That sounds good, I wasn't sure what the repo policy was on changing the version number.  I will go ahead and back out that change.

Reply:
Ok, the change has been backed out.